packages/babel-plugin-transform-wpcalypso-async/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
```

Comment:
If we're bumping to 2.0 in this PR, should we release those changes and add this under the new version? In other words, should we bump the version in the same PR where we update the CHANGELOG to be for 2.0?

Reply:
There is one practical reason to bump the version now: it will clear the Babel cache. The version field is used to invalidate the cache key:

https://github.com/Automattic/wp-calypso/blob/trunk/build-tools/babel/babel-loader-cache-identifier/index.js#L23-L25

I ran into this when working on the PR: I modified the transform but the dev server was still using the old one, even after restarting, because it was using cached transpiled files.

Reply:
Makes sense, thanks for elaborating!